enterprise/admin/views.py

```diff
@@ -264,6 +267,14 @@ def get_enterprise_customer_user_queryset(self, search_keyword, customer_uuid):
             matching_user_ids = matching_users.values_list('pk', flat=True)
             learners = learners.filter(user_id__in=matching_user_ids)
 
+        if all_learners is None:
```

Comment:
Can we combine this code with the condition above `if all_learners or search_keyword is None:` ?